concurrency/includes/2-channels.md

```diff
@@ -0,0 +1,460 @@
+# Communication Mechanisms
```

Comment:
@christianhxc can you think of a way to split this unit up? 460 lines total will take the learner maybe 15-20 minutes to finish, and that's a little long for one unit.

Reply:
@meaghanlewis oh right, I didn't notice the numbers of lines, let me change it and I'll let you know

Reply:
@meaghanlewis oh right, I didn't notice the number of lines, let me change it and I'll let you know

Reply:
@meaghanlewis done, let me know your thougths

Reply:
thanks for splitting this file up, looks good to me!